core/src/main/resources/lib/hudson/artifactList.properties

```diff
@@ -0,0 +1,26 @@
+# The MIT License
+#
+# Copyright 2021 CloudBees, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+building=\
+    The build is still in progress. \
+    Some artifacts are present but the list may be incomplete. \
+    Artifacts still being written may even be corrupt.
```

Comment:
File corruption could potentially be avoided if https://github.com/jenkinsci/jenkins/blob/52acfa962fcd118fe8a75bc9a80719a71485f743/core/src/main/java/hudson/FilePath.java#L2838-L2853 were reworked to use a temporary file + atomic move, but that could have farther-ranging effects. Anyway we already rendered the artifact list from direct navigation, exposed artifacts via REST, etc. for running builds and no one seems to have complained about this not being atomic.

Reply:
seems a bit weird to include the last line in a user facing message =/.

I would omit it

Reply:
Well, it could potentially happen to you that the build happens to be in the middle of archiving some 3Gb binary, and you see it listed on the artifacts page and innocently go to download it and get a partial file. Rather unlikely in practice but seems appropriate to note the possibility.